config/routes.rb

```diff
@@ -16,4 +16,8 @@
     resources :outreach_events, only: :index
     resources :outreach_event_types, except: [:show]
   end
+
+  namespace :facilitator do
+    resources :outreach_events, only: :index
+  end
```

Comment:
This shouldn't be necessary if we reuse the `outreach_events/index` view and either add logic to switch which outreach events list is outputted or consolidate the outreach events lists into one "smart" partial